app/models/rental.rb

```diff
@@ -29,6 +29,7 @@ class Rental < ActiveRecord::Base
   scope :upcoming_rentals, -> { reserved.where('start_time <= ? AND end_time >= ?', DateTime.current + 1.day, DateTime.current) }
   scope :all_future_rentals, -> { reserved.where('end_time >= ?', DateTime.current) }
   scope :no_show_rentals, -> { reserved.where('end_time < ?', DateTime.current) }
+  scope :inactive_rentals, -> { where('rental_status = ? OR rental_status = ?', 'canceled', 'checked_in') }
```

Comment:
Perhaps refactor `where(rental_status: [canceled, checked_in])`? And perhaps use the methods, if you have access to them, as opposed to strings